#To hold spline seed knot points for stick trajectory 
geometry_msgs/Point[] left_seed
geometry_msgs/Point[] right_seed
float32[] time_seed
